src/site/static/eligibility.liquid

```diff
@@ -1502,8 +1507,25 @@ pageClass: "page-public-assistance"
   }
 
   function feraEligible() {
-    // TODO
-    return true;
+    const grossLimit = MonthlyIncomeLimit.fromAnnual([
+      0,
+      0,  
+      57575,
+      69375,
+      81175,
+      92975,
+      104775,
+      116575
+    ],
+    11800);
+
+    return (
+      and(
+        gt(grossIncome(), careIncomeLimit()),
+        le(grossIncome(), grossLimit.getLimit(householdSize())),
+        householdSize() >= 3
```

Comment:
Generally it is good practice to define "magic numbers" like this `3` as a `const` variable.  That allows you to self-document the number with a descriptive variable name and add some comments further explaining the number.  So, you could add this below the grossLimit definition:

`const MIN_HOUSEHOLD_SIZE = 3;`

And change this line to `householdSize() >= MIN_HOUSEHOLD_SIZE`

**As an aside:** Note the different naming convention for a number that should be treated truly as an unchangeable constant. Just declaring something `const` in javascript does not prevent someone from altering it's value:
```
const myArr = [1,2,3];
myArr.push(4);
console.log(myArr);  // [1, 2, 3, 4]
```